servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -1639,6 +1645,57 @@ private TypeSpec newServiceInterfaceSpec(final State state, final boolean blocki
                 .addStatement("return $L", blocking ? BLOCKING_METHOD_DESCRIPTORS : ASYNC_METHOD_DESCRIPTORS)
                 .build());
 
+        // generate default service methods
+        if (defaultServiceMethods) {
+            final List<RpcInterface> interfaces = state.serviceRpcInterfaces.stream()
+                    .filter(intf -> intf.blocking == blocking)
+                    .collect(toList());
+            for (int i = 0; i < interfaces.size(); ++i) {
+                final RpcInterface rpcInterface = interfaces.get(i);
+                final MethodDescriptorProto methodProto = rpcInterface.methodProto;
+                final ClassName inClass = messageTypesMap.get(methodProto.getInputType());
+                final ClassName outClass = messageTypesMap.get(methodProto.getOutputType());
+                final String methodName = sanitizeIdentifier(methodProto.getName(), true);
+                final String methodPath = context.methodPath(state.serviceProto, methodProto).substring(1);
+                final int methodIndex = i;
+                final MethodSpec methodSpec = newRpcMethodSpec(inClass, outClass, methodName,
+                        methodProto.getClientStreaming(),
+                        methodProto.getServerStreaming(),
+                        !blocking ? EnumSet.of(INTERFACE) : EnumSet.of(INTERFACE, BLOCKING, SERVER_RESPONSE),
+                        printJavaDocs, (__, spec) -> {
+                            final String errorMessage = "\"Method " + methodPath + " is unimplemented\"";
+                            spec.addModifiers(DEFAULT).addParameter(GrpcServiceContext, ctx);
+                            spec.addAnnotation(Override.class);
+                            if (!blocking) {
+                                final ClassName returnType = methodProto.getServerStreaming() ? Publisher : Single;
+                                spec.addStatement("return $T.failed(new $T(new $T($T.UNIMPLEMENTED, $L)))",
+                                        returnType, GrpcStatusException, GrpcStatus, GrpcStatusCode, errorMessage);
+                            } else {
+                                if (!skipDeprecated && methodProto.getServerStreaming()) {
+                                    spec.addStatement("$T.super.$L(ctx, request, response)",
+                                            rpcInterface.className, methodName);
+                                } else {
+                                    spec.addStatement("throw new $T(new $T($T.UNIMPLEMENTED, $L))",
+                                            GrpcStatusException, GrpcStatus, GrpcStatusCode, errorMessage);
+                                }
+                            }
+                            if (printJavaDocs) {
```

Comment:
I realized that in this case we actually don't need to generate javadoc because these methods are overrides and javadoc is automatically inherited from the `*Rpc` interfaces:
```patch
Index: servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java b/servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java
--- a/servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java	(revision 9120c33cf95e6a1cf2cd8f482de3b51b108f13cc)
+++ b/servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java	(date 1732233569524)
@@ -1662,7 +1662,7 @@
                         methodProto.getClientStreaming(),
                         methodProto.getServerStreaming(),
                         !blocking ? EnumSet.of(INTERFACE) : EnumSet.of(INTERFACE, BLOCKING, SERVER_RESPONSE),
-                        printJavaDocs, (__, spec) -> {
+                        false, (__, spec) -> {
                             final String errorMessage = "\"Method " + methodPath + " is unimplemented\"";
                             spec.addModifiers(DEFAULT).addParameter(GrpcServiceContext, ctx);
                             spec.addAnnotation(Override.class);
@@ -1679,11 +1679,6 @@
                                             GrpcStatusException, GrpcStatus, GrpcStatusCode, errorMessage);
                                 }
                             }
-                            if (printJavaDocs) {
-                                extractJavaDocComments(state, methodIndex, spec);
-                                spec.addJavadoc(JAVADOC_PARAM + ctx +
-                                        " context associated with this service and request." + lineSeparator());
-                            }
                             return spec;
                         });

```